routes/demos.js

```diff
@@ -55,12 +55,19 @@ module.exports = function (express) {
           const sampleUrl = new URL(samplePath, tauExamplesHost);
           // Define number of unnecessary directories to be omitted while downloading.
           const numDirsToCut = sampleUrl.pathname.match(/TAU\/examples\/mobile|wearable\/UIComponents/g) ? 4 : 0;
```

Comment:
numDirsToCut is not needed here. Please move it below.

Reply:
done